src/gennylib/src/encryption.cpp

```diff
@@ -305,6 +332,55 @@ void FLEEncryptedField::appendEncryptInfo(sub_document subdoc) const {
     }
 }
 
+QueryableEncryptedField::Query::Query(const Node& yaml) {
+    queryType = yaml["queryType"].to<std::string>();
+    contention = yaml["contention"].maybe<int>();
```

Comment:
What value do we get when it's empty, 0?

Reply:
`contention` is a std::optional, so if there's no value we don't include the "contention" field in the schema. Server will interpret a missing "contention" field as the implicit default value of 4.

Reply:
I was wondering if there is a way to simply convert the yaml `Node` into JSON, and simply use `bsoncxx::from_json()` when appending it to the generated schema.  Since we're going to add more fields to the `query` node once QE range and substring search is implemented, it'll be nice to not have to parse and validate each field, only to simply serialize it back.  Also, this would allow for usage of extended JSON for complex types like decimal128. For instance, with "range" queryType, a queries node could look something like this:
```
queries: [ { queryType: "range", min: { $numberDecimal: "3.14" }, max: { $numberDecimal: "12.9" }, sparsity: 3 } ]
```

Reply:
That might be something to try, I think the testlib code does something similar [here](https://github.com/mongodb/genny/blob/master/src/testlib/include/testlib/helpers.hpp#L42-L46).

My only concern would be that the syntax visible to the user must remain the same.